pkg/volume/rbd/rbd_util.go

```diff
@@ -133,6 +132,17 @@ func rbdErrors(runErr, resultErr error) error {
 	return resultErr
 }
 
+// 'rbd' utility simply pass '-m <mon>' parameter to kernel rbd/libceph
```

Comment:
passes

Reply:
Nit: technically, the "-m" / "--mon_host" CLI optional is used for the CLI to connect to one or more monitors, grab the full monitor map, and it builds its own comma-separated list of IP addresses to send to the kernel.

Reply:
@rootfs Comments are updated in https://github.com/kubernetes/kubernetes/pull/57512/commits/4598f0df1ec74122218fd5d7082930f9264d5e83#diff-36f18e327f36d95eb1333c4a18781184L135.